multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -435,7 +489,11 @@ def infer_column_types(
         elif is_numerical_column(data[col_name], valid_data[col_name]):  # Infer numerical column
             column_types[col_name] = NUMERICAL
         elif is_image_column(data[col_name], col_name=col_name, image_type=IMAGE_PATH):  # Infer image-path column
-            column_types[col_name] = IMAGE_PATH
+            # Check if it is document image or not.
+            if is_document_image_column(data[col_name], col_name=col_name):
+                column_types[col_name] = DOCUMENT
```

Comment:
To distinguish from other document types, e.g., pdf, we may need be more specific about the document type, e.g., `doc_image`.

Reply:
add a new type: `document_image`